### ISSUE TITLE
http://juzzlin.github.io/Heimer displays 404

### ISSUE THREAD

**Issue by mkg20001** [juzzlin/Heimer#42]
It is linked from the about page

![screenshot](https://user-images.githubusercontent.com/7735145/50481406-9b576b80-09e1-11e9-9f56-907ed6a3548c.png)

**Comment by mkg20001** [comment]
Likely because the gh-pages branch is missing.
You can also let github pages use master as the main branch so it at least shows the README instead of a 404

**Comment by juzzlin** [comment]
Yeah, the webpage currently doesn't exist :) Thanks for reminding me.
